tuxemon/technique.py

```diff
@@ -260,7 +263,10 @@ def use(self, user: Monster, target: Monster) -> TechniqueResult:
         }
         # TODO: handle conflicting values from multiple technique actions
         # TODO: for example, how to handle one saying success, and another not?
-       
+        print(target.sleep)
+        if target.sleep > 0:
```

Comment:
probably not likely to happen, but if a technique has other effects (poison?), and the target is sleeping, then the other effects will never be processed.  if poison was applied separately, it would still take damage, i believe.